README.md

```diff
@@ -786,6 +787,23 @@ DeviceInfo.getIncremental().then(incremental => {
 
 ---
 
+### getInstallerPackageName()
+
+The internal value used by the underlying source control to represent this build.
+
+#### Examples
+
+```js
+DeviceInfo.getInstallerPackageName().then(installerPackageName => {
+  // Play Store: "com.android.vending"
+  // Amazon: "com.amazon.venezia"
+  // Samsung App Store: "com.sec.android.app.samsungapps"
+  // Developer, iOS: "null"
```

Comment:
```suggestion
  // Developer, iOS: "unknown"
```

(that's actually what you are already returning, and is in harmony with our stated "default" values for unsupported platforms)